Documentation/ExtensionArchitecture/BestPractises/NamingConventions.rst

```diff
@@ -220,6 +220,27 @@ Examples for Extbase domain models and table names of an extension named `cool_s
 | :php:`Vendor\CoolShop\Domain\Model\Billing\Address` | :sql:`tx_coolshop_domain_model_billing_address` |
 +-----------------------------------------------------+-------------------------------------------------+
 
+MM tables (for multiple-multiple relations between tables) follow these rules.
+
+Extbase tables SHOULD follow this pattern:
+
+.. code-block:: none
+
+   # rule for Extbase
+   tx_<extension-prefix>_domain_model_<model-name-1>_<model-name-2>_mm
+   # example: EXT:news with relation between news and tag
+   tx_news_domain_model_news_tag_mm
```

Comment:
Either use the above blogexample post scenario for consistency (relation between post and tag) or move the tip below before the new MM tables section directly under the cool_shop example.

Reply:
ok, will push another commit.